dev/backend-tools-tests/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM python:3.8-alpine
 LABEL zimfarm=true
 LABEL org.opencontainers.image.source https://github.com/openzim/zimfarm
 
-RUN apk add openssl gcc libffi-dev libc-dev
+RUN apk add openssl gcc libffi-dev libc-dev openssh-keygen
```

Comment:
I don't see the value of having this in this container…

Reply:
This is needed when setting a user SSH key to generate its PKCS8 : https://github.com/openzim/zimfarm/blob/ac18dd3738e0d386f80c52c10979319776746dc6/dispatcher/backend/src/routes/users/keys.py#L93

Reply:
OK ; I don't recall seeing tests for the SSH key endpoint.